v1.1/pvp-specification.md

```diff
@@ -146,14 +146,14 @@ Now consider this scenario:
     version number bumped, as required by the PVP.
 2.  Package B, which can still work with the old and new version of
     package A, changes its dependency on package A to allow for both
-    versions. Package B only gets a patch-level bump.
-3.  Package C might or might not compile, depending on which patch-level
+    versions. Package B only gets a minor version number bump.
```

Comment:
This is still wrong. "Minor version bump" semantics suggests there are *new features*. Compatibility patch (e.g. relaxing bounds, or even smal lcode changes like adding CPP for compatibility), doesn't require minor version bump.